libs-rs/build_all.sh

```diff
@@ -5,6 +5,9 @@ set -e
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-$DIR/eth-client/build.sh
-$DIR/eth-prover/build.sh
-$DIR/fungible-token/build.sh
+cd $DIR/eth-client
+./build.sh
```

Comment:
Found cargo doesn't actually build if don't cd, and the later cp target/.../eth-prover.wasm ../ still copy, so that build_all.sh was not very helpful